snippets/send_escrow.py

```diff
@@ -64,3 +66,41 @@
 print("Balances of wallets after Escrow was sent:")
 print(get_balance(wallet1.address, client))
 print(get_balance(wallet2.address, client))
+
+# Setup conditional escrows
+cryptoCondition = generate_escrow_cryptoconditions(urandom(32))
+
+create_tx = EscrowCreate(
+    account=wallet1.address,
+    destination=wallet2.address,
+    amount="1000000",
+    condition=cryptoCondition["condition"],
+    cancel_after=datetime_to_ripple_time(datetime.now()) + 100,
+)
+
+create_escrow_response = submit_and_wait(create_tx, client, wallet1)
```

Comment:
I have empirically observed that the generated cryptographic condition always begins with `A0258020...` and ends with `...810120`. I don't have statistical numbers to prove my point, but the outputs from this library do not appear random, despite varying secret inputs.